src/charts/CategoricalBarChart/index.tsx

```diff
@@ -95,6 +95,22 @@ export function CategoricalBarChart({
           text: title,
           left: "center",
         },
+        tooltip: {
+          trigger: "item",
+          formatter: (params) => {
```

Comment:
it was not possible to use formatter string used in pie chart
https://echarts.apache.org/en/option.html#grid.tooltip.formatter
because the value is of type object.